addons/account/models/account_lock_exception.py

```diff
@@ -0,0 +1,190 @@
+from odoo import _, api, fields, models, Command
+from odoo.tools import create_index
+from odoo.tools.misc import format_date
+from odoo.exceptions import UserError
+
+from odoo.addons.account.models.company import SOFT_LOCK_DATE_FIELDS
+
+
+class AccountLockException(models.Model):
+    _name = "account.lock_exception"
+    _description = "Account Lock Exception"
+
+    active = fields.Boolean('Active', default=True)
+    state = fields.Selection(
+        selection=[
+         ('active', 'Active'),
+         ('revoked', 'Revoked'),
+         ('expired', 'Expired'),
```

Comment:
No reason not to use 4 spaces here
```suggestion
            ('active', 'Active'),
            ('revoked', 'Revoked'),
            ('expired', 'Expired'),
```